### ISSUE TITLE
Rename/move doesn't work on Windows

### ISSUE THREAD

**Issue by movingobjects** [movingobjects/bwco-asset-downloader#5]
Running the downloader on Windows returns this problem:

```
fs.js:766
  return binding.rename(pathModule._makeLong(oldPath),
                 ^

Error: EPERM: operation not permitted, rename...
```

**Comment by movingobjects** [comment]
I've tried using `fs-extra`'s `moveSync` and `fs`'s `renameSync` ... both have the same issue

**Comment by movingobjects** [comment]
Fixed a74e2a21e9358bf42d7edc732b5d0b697b0f07fd
